api/rethinkdb.js

```diff
@@ -0,0 +1,10 @@
+import rethinkdbdash from 'rethinkdbdash'
+import {rethinkdb} from './config'
```

Comment:
Perhaps skip config for this, and use the new .env entries directly in this file.